nautobot-app/{{ cookiecutter.project_slug }}/docs/requirements.txt

```diff
@@ -1,5 +1,6 @@
-mkdocs==1.5.2
-mkdocs-material==9.1.15
-mkdocs-version-annotations==1.0.0
-mkdocstrings-python==1.5.2
-mkdocstrings==0.22.0
+mkdocs==1.6.0
+mkdocs-material==9.5.32
+markdown-version-annotations==1.0.1
+griffe==1.1.1
+mkdocstrings-python==1.10.8
+mkdocstrings==0.25.2
```

Comment:
As discussed, lets just pin griffe to whatever version is compatible with the existing dependencies. If that's not possible lets go ahead and update everything but make sure we coordinate with the core team